test/goss/remote/goss.yaml

```diff
@@ -0,0 +1,3 @@
+gossfile:
```

Comment:
It's the default checks file which `goss` runs if not otherwise specified with `-g`. It can contain multiple gossfiles. I'll remove it as it's not used